src/FM.LiveSwitch.Mux/FM.LiveSwitch.Mux.csproj

```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.10" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.10" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="FM.LiveSwitch" Version="1.10.0.34671" />
```

Comment:
This is not right. Why did we add this dependency?

Reply:
This dependency is needed to use Matroska library to measure media file duration.